README.md

```diff
@@ -6,6 +6,38 @@ Create a community of like minded authors to foster inspiration and innovation
 by leveraging the modern web.
 
 ---
+## Getting started 
+These instructions will get you a copy of the project up and running in your local machine for development and testing purposes.
+
+## Prerequisites
+- [Git](https://git-scm.com/download/)
+- [Python 3 and above](https://www.python.org/downloads/)
```

Comment:
We will need to use versions 3.6 or higher, state it here

Reply:
There are other versions of Python 3 which we do not want, like 3.3, 3.4, 3.5 because we might use some functions only available with version 3.6 and higher.